### ISSUE TITLE
Polymer version

### ISSUE THREAD

**Issue by chasb154** [webcomponents/webcomponents.org#980]
[Origin page](https://www.webcomponents.org/search/dialog)

I'm looking for elements but I've switched over to Polymer 2.0. I need to be able to see Polymer version on the list. I find myself opening every element on my search results and closing the ones that are Polymer 1.0.

It would be great to see Polymer version on returned search results.

**Comment by seaneking** [comment]
Sets a dangerous tie-in precedent with Polymer - what about elements not using Polymer? Or a WC framework at all? Think this would be better addressed with tags (making them more prominent? Documenting the pattern for component authors to follow?) or a general `dependency:name#version` or similar search operator

**Comment by chasb154** [comment]
Good Point. As the web components become more broadly used there will be more varied dependency requirements. Right now I'm only concerned about dependencies on Polymer but that's my choice due to the ecosystem and support for it. 

Your idea for tags is better and, I think, going to be needed soon if web components continue to grow.

**Comment by jaakidup** [comment]
What is the latest idea regarding this issue? 
It is becoming a real thing to only find compatible components

**Comment by stramel** [comment]
Maybe having a libraries filter?

- Native
- Polymer 1
- Polymer 2
- other libs

These could probably be populated based on what libraries and versions the components in the catalog depend on?

**Comment by Link2Twenty** [comment]
Would filter by dependencies be best?

**Comment by chasb154** [comment]
Dependencies would work very well.

**Comment by chasb154** [comment]
Hit close by mistake.
